libknet/libknet.h

```diff
@@ -153,6 +159,57 @@ knet_handle_t knet_handle_new(knet_node_id_t host_id,
 
 int knet_handle_free(knet_handle_t knet_h);
 
+/**
+ * knet_handle_set_threads_timer_res
+ * @brief Change internal thread timer resolutions
+ *
+ * knet_h   - pointer to knet_handle_t
+ *
+ * timeres  - some threads inside knet will use usleep(timeres)
+ *            to check if any activity has to be performed or wait
+ *            for the next cycle. The timeres (expressed in nano seconds)
+ *            define this interval, with a default of KNET_THREADS_TIMER_RES
```

Comment:
... for the next cycle. 'timeres' (expressed in nanoseconds) defines this interval ...